x-pack/plugins/security_solution/public/flyout/document_details/right/components/about_section.tsx

```diff
@@ -36,11 +66,17 @@ export const AboutSection: VFC<AboutSectionProps> = ({ expanded = true }) => {
         />
       }
       data-test-subj={ABOUT_SECTION_TEST_ID}
+      gutterSize="s"
     >
-      <Description />
-      <EuiSpacer size="m" />
-      <Reason />
-      <MitreAttack />
+      {eventKindInECS &&
+        (eventKind === 'event' ? (
+          // if event kind is event, show a detailed description based on event category
+          <EventCategoryDescription />
+        ) : (
+          // if event kind is not event, show a higher level description on event kind
+          <EventKindDescription eventKind={eventKind} />
```

Comment:
I go back and forth here, thinking that we could merge these 2 components. What you have is good, but for some reason I feel like having that distinction is too much for this high level `AboutSection` component... Just a thought...
I feel like having simply an `EventDescription` here would be simpler. The logic of switching could be done within in.

Reply:
I prefer to keep it as is until the event logic becomes more involved. Another reason being, after updating the documentation, it is more clear to me the relationship of these conditions
```
/**
 * Most top section of the overview tab.
 * For alerts (event.kind is signal), it contains the description, reason and mitre attack information.
 * For generic events (event.kind is event), it shows the event category description and event renderer.
 * For all other events, it shows the event kind description, a list of event categories and event renderer.
 */
```